Dockerfile

```diff
@@ -4,4 +4,4 @@ LABEL authors="phil.ewels@scilifelab.se" \
 
 COPY environment.yml /
 RUN conda env create -f /environment.yml && conda clean -a
-ENV PATH /opt/conda/envs/nf-core-rnaseq-1.3/bin:$PATH
```

Comment:
@jburos Thanks for the PR. It looks like you have cloned and edited the code on the current `master` branch (`1.3`). You will have to make the edits on the `dev` branch (`1.4dev`) please. You wont need to increment the version number anywhere because this will already be the developmental version for the next release 👍